stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -348,6 +348,56 @@ public struct Unsafe${Mutable}RawBufferPointer
     return 0
   }
 
+  %  if mutable:
+    /// Initializes memory in the buffer with the elements of
+    /// `source` and binds the initialized memory to type `T`.
+    ///
+    /// Returns an iterator to any elements of `source` that didn't fit in the 
+    /// buffer, and an index to the point in the buffer one past the last element
+    /// written.
```

Comment:
Does it make more sense to give this in terms of bytes or elements? Currently looks like bytes, but generally speaking RawPointers seem to have a philosophy of "you initialized me, now I have this type". The right answer probably depends on what the "expected" code that uses this API is.